backend/entityservice/settings.py

```diff
@@ -33,6 +33,7 @@ class Config(object):
     DATABASE = os.getenv('DATABASE', 'postgres')
     DATABASE_USER = os.getenv('DATABASE_USER', 'postgres')
     DATABASE_PASSWORD = os.getenv('DATABASE_PASSWORD', '')
+    DATABASE_MAX_CONNECTIONS = os.getenv('DATABASE_MAX_CONNECTIONS', '5')
```

Comment:
is that a good default value? How many concurrent connections do we typically have in a worker?

Reply:
Workers need only 1. But flask sometimes need more. I will need to check what is returned by the server when it happens. A 500 would be good, asking the client to retry.

Reply:
Tested, it return well a 500. Returning a 503 would be better, but that's a good start.